lib/sm/client.rb

```diff
@@ -374,5 +381,19 @@ def validate_reply_context(args)
         raise Common::Exceptions::ValidationErrors, draft
       end
     end
+
+    ##
+    # @!group StatsD
+    ##
+    # Report stats of secure messaging events
+    #
+    def statsd_cache_hit
+      StatsD.increment("#{STATSD_KEY_PREFIX}.cache.hit")
+    end
+
+    def statsd_cache_miss
+      StatsD.increment("#{STATSD_KEY_PREFIX}.cache.miss")
+    end
```

Comment:
Minor nitpick but you could make these private if we don't expect them to be called from anywhere else